### ISSUE TITLE
mse loss on 1/(2*beta) or 0.5 * beta?

### ISSUE THREAD

**Issue by Alaya-in-Matrix** [tengxiao1/Cal-DPO#1]
In Eq. 10 of the Cal-DPO paper, the mse loss is about log ratio and 1/(2 * beta), however in the pseudocode from Appendix B.1, you have `F.mse_loss(chosen_reward, 0.5*beta)`, which is inconsistent to Eq. 10. Which one should I use?

**Comment by tengxiao1** [comment]
Hi, Equation (10) is correct. We made a typo in Appendix and we will correct the typo in the revision.
